packages/widgets/src/media-transcoding/panels/queue.panel.tsx

```diff
@@ -0,0 +1,64 @@
+import { Center, Group, ScrollArea, Table, Text, Title, Tooltip } from "@mantine/core";
+import { IconHeartbeat, IconTransform } from "@tabler/icons-react";
+
+import { humanFileSize } from "@homarr/common";
+import type { TdarrQueue } from "@homarr/integrations";
+import { useI18n } from "@homarr/translation/client";
+
+interface QueuePanelProps {
+  queue: TdarrQueue;
+}
+
+export function QueuePanel(props: QueuePanelProps) {
+  const { queue } = props;
+
+  const t = useI18n("widget.mediaTranscoding.panel.queue");
+
+  if (queue.array.length == 0) {
+    return (
+      <Center style={{ flex: "1" }}>
+        <Title order={3}>{t("empty")}</Title>
+      </Center>
+    );
+  }
+
+  return (
+    <ScrollArea style={{ flex: "1" }}>
+      <Table style={{ tableLayout: "fixed" }}>
+        <thead>
+          <tr>
+            <th>{t("table.file")}</th>
+            <th style={{ width: 80 }}>{t("table.size")}</th>
+          </tr>
+        </thead>
+        <tbody>
```

Comment:
Is there a reason you use the default html elements and not `Table.THead` and such? Then you could also use `w={80}`